locales/pt/json.json

```diff
@@ -89,7 +89,7 @@
     "Bahrain": "Barém",
     "Bangladesh": "Pai",
     "Barbados": "Barbados",
-    "Before continuing, could you verify your email address by clicking on the link we just emailed to you? If you didn't receive the email, we will gladly send you another.": "Before continuing, could you verify your email address by clicking on the link we just emailed to you? If you didn't receive the email, we will gladly send you another.",
+    "Before continuing, could you verify your email address by clicking on the link we just emailed to you? If you didn't receive the email, we will gladly send you another.": "Antes de continuar, poderia verificar o seu e-mail ao clicar no link que lhe acabámos de enviar por e-mail? Se não recebeu o e-mail, enviar-lhe-emos de bom grado outro.",
```

Comment:
I don't know if I agree with the usage of "clicando" in this sentence. This because in PT-PT we don't often use the gerund when comunicating but languages is not my strongest and maybe I am wrong.